drivers/mbox/CMakeLists.txt

```diff
@@ -7,4 +7,5 @@ zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_USERSPACE   mbox_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_NRFX_IPC   mbox_nrfx_ipc.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_NXP_S32_MRU   mbox_nxp_s32_mru.c)
+zephyr_library_sources_ifdef(CONFIG_MBOX_NXP_FSL_MU   mbox_nxp_fsl_mu.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_ANDES_PLIC_SW   mbox_andes_plic_sw.c)
```

Comment:
Here should be `mbox_nxp_imx_mu` instead of `mbox_nxp_fsl`...

Reply:
To clarify, would you like the .c file to be named `mbox_nxp_imx_mu.c`, and the Kconfig to be named `CONFIG_MBOX_NXP_IMX_MU`, and all the driver functions to be prefixed with `nxp_imx_mu_`?

Reply:
Yes.